Sun Nov  4 18:32:13 FET 2012  Sergei Trofimovich <slyfox@community.haskell.org>
  * allow cmdargs-0.10
diff -rN -u old-diagrams-cairo/diagrams-cairo.cabal new-diagrams-cairo/diagrams-cairo.cabal
--- old-diagrams-cairo/diagrams-cairo.cabal	2012-11-04 18:32:51.542380791 +0300
+++ new-diagrams-cairo/diagrams-cairo.cabal	2012-11-04 18:32:51.543380791 +0300
@@ -35,7 +35,7 @@
                        diagrams-core >= 0.5 && < 0.6,
                        diagrams-lib >= 0.5 && < 0.6,
                        cairo >= 0.10.1 && < 0.13,
-                       cmdargs >= 0.6 && < 0.10,
+                       cmdargs >= 0.6 && < 0.11,
                        gtk >= 0.12.0  && < 0.13.0,
                        split >= 0.1.2 && < 0.3
   if !os(windows)
